.github/ISSUE_TEMPLATE/editorial-change--.md

```diff
@@ -0,0 +1,9 @@
+---
+name: "Editorial Change"
+about: Proposal for an editorial change to the WebAuthn specification
+labels: ["type:editorial"]
+---
+
+## Proposed Change
+
+Describe your proposed change 
```

Comment:
Possibly hint "an editorial change is one that doesn't affect the behavior of the spec. If your proposed change would affect behavior, even if minor, please file as technical."

Reply:
and "If you have suggested text, please file a corresponding Pull Request."

Reply:
Done. Thank you!